keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/probabilistic/KDEVertex.java

```diff
@@ -103,7 +104,7 @@ public DoubleTensor sample(int nSamples, KeanuRandom random) {
     }
 
     @Override
-    public DoubleTensor sample(KeanuRandom random) {
+    public DoubleTensor sampleWithShape(long[] shape, KeanuRandom random) {
         return sample(1, random);
```

Comment:
It seems wrong that this one doesn't use `shape`. Any thoughts on this @GeorgeNash / @gordoncaleb ?

Reply:
I believe this was a mistake with my find + replace

Reply:
The `GaussianKDE` algorithm asserts that the shape of the samples must be scalar. The `KDEVertex` provides a method `sample(int, KeanuRandom)` which returns a tensor of shape [1, n]. I therefore think that `KDEVertex` shouldn't implement `SamplableWithManyScalars`. I've changed it to implement `Samplable` instead.